src/authorization/authorization.ts

```diff
@@ -52,20 +52,20 @@ const doesRequestUrlContain = (routeConfig: RouteConfig) =>
   );
 export interface WithJWT {
   clearRefresh: () => void;
-  setEmail: (email: string) => Promise<string>;
-  setUserID: (userId: string) => Promise<string>;
+  setEmail: (email: string, identityResolution?: IdentityResolution) => Promise<string>;
+  setUserID: (userId: string, identityResolution?: IdentityResolution) => Promise<string>;
   logout: () => void;
   refreshJwtToken: (authTypes: string) => Promise<string>;
-  toggleAnonUserTrackingConsent: (consent: boolean) => void;
+  setVisitorUsageTracked: (consent: boolean) => void;
 }
 
 export interface WithoutJWT {
   setNewAuthToken: (newToken?: string) => void;
   clearAuthToken: () => void;
-  setEmail: (email: string) => Promise<void>;
-  setUserID: (userId: string) => Promise<void>;
+  setEmail: (email: string, identityResolution?: IdentityResolution) => Promise<void>;
+  setUserID: (userId: string, identityResolution?: IdentityResolution) => Promise<void>;
```

Comment:
Do we need to update implementations to use identityResolution?

Reply:
They already do! Was testing with Brad and the type in the example app didn't have access to pass in this param.